### PR TITLE
Add nano docs

### LINKED ISSUE #396: Create metadata schema for nanoDESI/nanoPOTS
The nanoDESI/nanoPOTS metadata schema is here in this google sheet with example input:

https://docs.google.com/spreadsheets/d/1q4nEZ1Ip83G6EUAkwCVWKPeeNhTgVEpKG7tANsgFAJg/edit#gid=1409865421

Note: ms_source & polarity fields can hold any combination of nanoDESI/nanoPOTS & positive/negative ion mode

### PR DESCRIPTION
**Pull request by mccalluc**
Fix #396

### CODE REVIEW

File: docs/nano/README.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+| constraint | value |
+| --- | --- |
+| required | `True` |
+
+### `polarity`
+The polarity of the mass analysis (positive or negative ion modes)
+
+| constraint | value |
+| --- | --- |
+| required | `True` |
```
Comment (mccalluc) [inline docs/nano/README.md:184]: @cebriggs7135 : I'm guessing both of these should have enum constraints?

Comment (mccalluc) [reply docs/nano/README.md]: pushed enums, both for nano and maldi.

**Comment by mccalluc** [comment]
@cebriggs7135 : Noticing that maldiims.yaml has
```yaml
  -
    name: assay_type
    constraints:
      enum:
        - MALDI-IMS positive
        - MALDI-IMS negative
```
and
```yaml
  -
    name: polarity
    description: The polarity of the mass analysis (positive or negative ion modes)
    constraints:
      enum:
        - negative ion mode
        - positive ion mode
```
Could we change the Assay type to plain `MALDI-IMS`... or is it meaningful to have `MALDI-IMS negative` with `positive ion mode`?

**Comment by cebriggs7135** [comment]
Yes, there can be just one name for MALDI. The polarity field takes care of the negative v positive mode. Same for nanoDESI/POTS.

**Review by cebriggs7135 — APPROVED** [review]
Yes, just one MALDI assay will suffice and the "polarity" field accommodates positive v negative mode.

**Comment by mccalluc** [comment]
One travis run stalled. Going ahead with merge.

**Pull request merged**